build_tools/benchmarks/common/benchmark_definition.py

```diff
@@ -443,7 +443,7 @@ def get_aggregate_time(self, benchmark_index: int, kind: str) -> int:
       if bench_case["name"].endswith(f"real_time_{kind}"):
         if bench_case["time_unit"] != "ms":
           raise ValueError(f"Expected ms as time unit")
```

Comment:
We could consider adding real support for other time units. Nanoseconds is actually the default. There's a comment about the choice of milliseconds here: https://github.com/iree-org/iree/blob/4a1286029910a2e67f4acecf186db640c8929d59/tools/iree-benchmark-module-main.cc#L188 (likely authored by yours truly).

Reply:
Curious about the problems of using floats : )

Another issue is that the dashboard doesn't support the unit, so we either need to rewrite the database into nanoseconds or make some changes to store unit in the database.

Reply:
> Another issue is that the dashboard doesn't support the unit, so we either need to backfill the database into nanoseconds or make some changes to store the unit info in the database.

Yup, we have full control over the dashboard website and its stack. We can expose units in whatever way we see suitable. :) 

Right now we have one single unit and that's [printed](https://github.com/iree-org/iree/blob/main/build_tools/benchmarks/upload_benchmarks_to_dashboard.py#L36-L44) at the beginning of the page for each benchmark series. We can parameterize that to use different units. This is simpler as we just need to handle the unit stuff at the IREE side. But it does have issue that the unit is not directly adjacent to the graph. 

If we want to show on the graph as axis label, that's also doable. The graph is rendered using [dygraph](https://dygraphs.com/), which supports using `ylabel` to set the label for y axis. Just need to plumb such unit information through to [here](https://github.com/google/dana/blob/main/www/public/js/uilibOne.js#L516). We can put the unit as an additional field to [`addSerie`](https://github.com/google/dana/blob/main/docs/Apis.md#addserie).